keyboards/rico/phoenix_project_no1/info.json

```diff
@@ -0,0 +1,123 @@
+{
+    "manufacturer": "Rico",
+    "keyboard_name": "Phoenix Project No 1",
+    "maintainer": "Rico",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP24", "GP0", "GP14", "GP13", "GP12", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP25", "GP2", "GP1"],
+        "rows": ["GP27", "GP29", "GP19", "GP20", "GP26"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 16
+    },
+    "url": "https://github.com/mymakercorner/Phoenix_Project_No1",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x9901",
+        "vid": "0x21C0"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP28"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This looks to be a 65_ansi layout, and the community layout suppot would be ideally.

Reply:
Yes it is 65%, but LAYOUT is a combination of all possible layouts, no only ANSI (there is ISO support as well, 6.25/7u space and split backspace).

Do you want me to add other layouts, like LAYOUT_65_ansi or LAYOUT_ansi (and maybe LAYOUT_iso) ?
I'd then keep LAYOUT and rename it LAYOUT_all (needed for VIA support).

Doing all possible combinations would be quite a long task, please tell me what your would like to see.

Reply:
Would love to see 65 ansi/iso support for it (and `LAYOUT_all`), but not necessarily all possible combinations. but that's not a hard requirement, IMO.  It just makes for a better experience for potential users to be able to use the community layout feature.

Reply:
I think I could add one layout for ansi (the one that is natively supported by the hotswap PCB) and another one for iso.
This would at least allow me to declare LAYOUT_all and ease potential future community layouts.

Reply:
That sounds fine.